testdata/loader-eb.elf

```diff
@@ -0,0 +1 @@
+loader-clang-12-eb.elf
```

Comment:
Do we want to commit these to the repo?

Reply:
I think it's useful to have a symlink to whatever the version compiled with the latest clang is. The other option is to hardcode `loader-clang-12-eb.elf` in the tests and change them when we upgrade clang version.

Reply:
I don't mind either way, up to you.

Reply:
Nope, I prefer the symlink approach. :+1: